CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Add tests for new FIM audit buffer option. ([#3990](https://github.com/wazuh/wazuh-qa/pull/3990)) \- (Framework + tests)
```

Comment:
```suggestion
- Add tests for new FIM audit buffer option. ([#4227](https://github.com/wazuh/wazuh-qa/pull/4227)) \- (Framework + tests)
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4227/commits/6e184c515b7af59320972a8e85d20f57dc7680f6